<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculsStarlink</class>
 <widget class="QFrame" name="CalculsStarlink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_groupe">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_groupe">
      <property name="text">
       <string extracomment="Starlink group">Groupe :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="groupe"/>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_starlink">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>761</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="parametrageDefaut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>128</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Paramétrage par défaut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculs">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>128</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Calculs</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_starlink1">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout_starlink1">
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_pasGeneration">
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Pas de génération :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_lieuxObservation">
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Lieu d'observation :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="pasGeneration">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>115</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string notr="true">1 seconde</string>
         </property>
         <item>
          <property name="text">
           <string>1 seconde</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 secondes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 secondes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20 secondes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30 secondes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 minute</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 minutes</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_nbJoursGeneration">
         <property name="text">
          <string>Nombre de jours de génération :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="nbJoursGeneration">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>115</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="lieuxObservation">
         <property name="minimumSize">
          <size>
           <width>195</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_hauteurSat">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Hauteur minimale du satellite :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_hauteurSoleil">
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Hauteur du Soleil :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <widget class="QComboBox" name="hauteurSat">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>137</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>137</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>136</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string notr="true">0°</string>
          </property>
          <item>
           <property name="text">
            <string>0°</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5°</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10°</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15°</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20°</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Autre...</string>
           </property>
          </item>
         </widget>
         <widget class="QLineEdit" name="valHauteurSat">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>0</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="inputMask">
           <string notr="true">99</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
         </widget>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QComboBox" name="hauteurSoleil">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>195</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string notr="true">Horizon (0°)</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="iconSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Horizon (0°)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Crépuscule civil (-6°)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Crépuscule nautique (-12°)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Crépuscule astronomique (-18°)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string extracomment="Pas de hauteur limite pour le Soleil : les calculs de jour sont inclus">Indifférent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Autre...</string>
           </property>
          </item>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <widget class="QLineEdit" name="valHauteurSoleil">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>74</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="inputMask">
         <string notr="true">###</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_lancement">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_lancement">
       <property name="text">
        <string extracomment="Starlink launch">Lancement :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lancement">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_deploiement">
       <property name="text">
        <string extracomment="Starlink deployment">Déploiement :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="deploiement">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
